tcf_website/static/reviews/club_review.js

```
@@ -0,0 +1,375 @@
+
+    // Now load semesters since a club is selected
+    loadSemesters();
+  });
+
+  // Fetch club details to get its category, then trigger category selection
+  function fetchClubDetails(clubId) {
+    $.ajax({
+      url: `/api/clubs/${clubId}/`,
+      dataType: "json",
+      success: function (club) {
+        if (club && club.category) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)